core/integration/storagetest/mutation_logs.go

```diff
@@ -80,13 +81,15 @@ func (mutationLogsTests) TestReadLog(ctx context.Context, t *testing.T, newForTe
 		{limit: 4, want: 6},    // Reading 4 items gets us into the second batch of size 3.
 		{limit: 100, want: 30}, // Reading all the items gets us the 30 items we wrote.
 	} {
-		rows, err := m.ReadLog(ctx, directoryID, logID, minWatermark, time.Now(), tc.limit)
-		if err != nil {
-			t.Fatalf("ReadLog(%v): %v", tc.limit, err)
-		}
-		if got := len(rows); got != tc.want {
-			t.Fatalf("ReadLog(%v): len: %v, want %v", tc.limit, got, tc.want)
-		}
+		t.Run(fmt.Sprintf("%d", i), func(t *testing.T) {
```

Comment:
I feel like I saw Pavel use an empty string instead of the `fmt.Sprintf` description and the framework automatically generates a description?

Reply:
Hmm, I suppose that's true.  For tests with the same name, the framework will prepend `_$i` to the name.  That feels a bit hacky. Here I preserve an explicit link between the test name and the row it's testing.